### ISSUE TITLE
Ad-blocker not working on Microsoft Edge for Linux

### ISSUE THREAD

**Issue by tanmay1024** [ghostery/ghostery-extension#1028]
### Description

Ad-blocker not working on Microsoft Edge for Linux

### Expected Behavior

Ads shouldn't show up whenever I open youtube or any video on it.

### Actual Behavior

Ads are showing up possibly everywhere on youtube.
Tried reinstalling Ghostery, rebooting browser and system, turning OFF and ON the Ad-Blocker. 
Nothing works.

### Versions

+ Browser: Microsoft Edge
+ OS: Ubuntu

**Comment by tanmay1024** [comment]
I have noticed that it has now also stopped working on Chrome for linux. Specifically on youtube.

**Comment by chapshaun** [comment]
Confirmed it's not working on Edge for me too, but I'm on Windows. 

Tried: uninstalling and reinstalling, turning off and on as well

**Comment by callie519** [comment]
when I try to open ghostery, the page is blank.

**Comment by GRadziejewski** [comment]
@callie519 is this blank page looks like this issue described here: #1151 ??

**Comment by callie519** [comment]
> @callie519 is this blank page looks like this issue described here: #1151 ??
Yes, it's exactly like that. I've tried reinstalling it several times but alas, no luck.

**Comment by callie519** [comment]
I did as others have suggested and downloaded it from the google chrome store and it works perfectly. Thank you👍👍
